packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -88,6 +107,15 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     disableDraggingSelection: false,
     clickSensitivity: 200,
     resetOnEnd: false,
+    scrubberInitialState: null,
+  };
+
+  componentDidMount = () => {
+    window.addEventListener('mouseup', this.handleDragEnd);
+  };
+
+  componentWillUnmount = () => {
+    window.removeEventListener('mouseup', this.handleDragEnd);
   };
 
   componentDidUpdate(prevProps: BaseBrushProps) {
```

Comment:
I think ideally we would update `state` if the `initialBrushState` updates, but don't want to add a ton of checks in this method. I guess for now users could force re-render with a new component `key` 🤔 not sure if @hshoff has thoughts here.

Reply:
This is a tough one... I've been thinking about this as well.

For now, I think pairing `key` with `initialBrushState` makes the most sense in this context. - It avoids the hazards of getting into "semi-controlled component" territory where users would _re_-default initial values but the component would otherwise manage its own logic.
- The behavior would be consistent with conventions around setting default values on other input elements (dropdowns, date pickers, HTML inputs, etc.) 

The downside I see is this wouldn't support animating between changes. My own immediate use case for this is for linking to different regions of a chart, so that would definitely be nice to have, but like you said, I would like to avoid adding checks everywhere...